aws/offsite/lambda/verify-database-copy/index.js

```diff
@@ -0,0 +1,114 @@
+console.log('Loading function');
+
+const AWS = require('aws-sdk');
+const mysqlPromise = require('promise-mysql');
+const Slack = require('slack-node');
+const promisify = require('util-promisify');
+const setTimeoutPromise = promisify(setTimeout);
```

Comment:
Why does this work? https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_promisify_original suggests that setTimeout would need to take a node-style callback function as its last argument, but the last argument to setTImeout is actually the delay (or possibly one of the optional params that gets passed to its callback): http://mdn.beonex.com/en/DOM/window.setTimeout.html

Reply:
Looks like `util-promisify` detects that it was passed the `setTimeout` function and re-orders the arguments and the callback:  https://github.com/juliangruber/util-promisify/blob/c10bf31fac99dfb7385834c96b96d3df0b2ebfde/index.js#L20-L33